capa/features/__init__.py

```diff
@@ -194,7 +198,7 @@ class StringFactory(object):
     def __new__(self, value, description=None):
         if value.startswith("/") and (value.endswith("/") or value.endswith("/i")):
             return Regex(value, description=description)
-        return String(value, description=description)
+        return String(str(codecs.decode(value, "unicode_escape")), description=description)
```

Comment:
decode string features as they are parsed.

e.g.

```yaml
- string: hello\r\nworld
```

will be internally represented as 

```
hello
world
```

versus

```
hello\r\nworld
```

allowing capa to properly match on features extracted from a given binary.

Reply:
nice!

why is there a wrapping `str(...)`?